queue.tf

```diff
@@ -28,3 +28,26 @@ resource "azurerm_key_vault_secret" "notifications_queue_listen_conn_str" {
   name         = "notifications-queue-listen-connection-string"
   value        = "${module.notifications-queue.primary_listen_connection_string}"
 }
+
+module "notifications-staging-queue" {
+  source              = "git@github.com:hmcts/terraform-module-servicebus-queue?ref=master"
+  name                = "notifications-staging"
+  namespace_name      = "${module.queue-namespace.name}"
+  resource_group_name = "${azurerm_resource_group.reform_scan_rg.name}"
+  lock_duration       = "PT5M"
+
+  duplicate_detection_history_time_window = "PT15M"
+  queue_enabled       = "${var.enable_staging_queue}"
```

Comment:
why not `count`?

Reply:
we need secrets for all env.
code will always bootstrap it

Reply:
does `queue_enabled` work on module ?

Reply:
For secrets condition seems fine but for module I think you will need to use count ?

Reply:
count is in the module which checks 'queue_enabled'

Reply:
What does the introduction of this parameter change compared to using count?

Reply:
https://github.com/hmcts/terraform-module-servicebus-queue/pull/3